README.md

```diff
@@ -1,16 +1,51 @@
-## Hi there 👋
-
-<!--
-**mfazekas/mfazekas** is a ✨ _special_ ✨ repository because its `README.md` (this file) appears on your GitHub profile.
-
-Here are some ideas to get you started:
-
-- 🔭 I’m currently working on ...
-- 🌱 I’m currently learning ...
-- 👯 I’m looking to collaborate on ...
-- 🤔 I’m looking for help with ...
-- 💬 Ask me about ...
-- 📫 How to reach me: ...
-- 😄 Pronouns: ...
-- ⚡ Fun fact: ...
--->
+# Hi, I'm Miklós Fazekas 👋
+
+## 🚀 About Me
+
+Senior Software Engineer with 20+ years of experience in mobile, web, and systems development. M.Sc. in Computer Science from Eötvös Loránd University. Currently serving as a maintainer of [@rnmapbox/maps](https://github.com/rnmapbox/maps), one of the most popular mapping libraries for React Native.
+
+## 💼 Current Work
+
+**Mobile Developer** *(Aug 2024 - Aug 2025)*  
+Working on **Zabit mobile app** - A cutting-edge mobile application built with Expo and React Native, featuring a Supabase backend for robust data management and scalable infrastructure.
+
+## 💼 Professional Experience
+
+- **Vice President** at [Morgan Stanley](http://www.morganstanley.com) - C++ developer working on proprietary middleware server for distributing market data (low latency/high throughput publish-subscribe system)
+- **Senior Software Engineer** at various technology companies
+- **iOS Development** experience with Objective-C/Cocoa/CocoaTouch
+- **Backend & Systems** development with C++, Ruby/Rails, Python
+- **Architecture & Infrastructure** design for large-scale applications
+- **Cross-platform Development** across mobile and desktop platforms
+
+## 🛠️ Technical Expertise
+
+- **Expert**: C++, Framework Design, Large Projects, Legacy Code
+- **Advanced**: Ruby/Rails, Python, TCP/IP Programming, React Native
+- **Intermediate**: Objective-C/iOS, Java, JavaScript/HTML/CSS, Expo
+- **Specialized**: Mapping & Geospatial (Mapbox, GIS), Multi-platform Development, Supabase
+- **Infrastructure**: CI/CD, Build Systems, Unit Testing, Git
+
+## 🗺️ Open Source Contributions
+
+- **[@rnmapbox/maps](https://github.com/rnmapbox/maps) Maintainer**: Leading React Native mapping solutions
+- **[React Native Core](https://github.com/facebook/react-native)**: Multiple merged PRs including Swift Package Manager support, Android autolinking fixes, and codegen improvements
+- **React Native Ecosystem**: Bug reproductions, feature implementations, and community contributions ([bug reports & PRs](https://github.com/search?q=author%3Amfazekas+react-native))
+- **[Expo Contributions](https://github.com/search?q=author%3Amfazekas+expo)**: Bug reproductions and improvements for Expo ecosystem
```

Comment:
Add link to https://github.com/expo/expo/issues/32073 and https://github.com/expo/expo/pull/32096 - fixing race condition in expo updates causing crash in permission request dialog